lib/ansible/modules/net_tools/dnsmadeeasy.py

```diff
@@ -681,16 +692,16 @@ def main():
         # create record and monitor as the record does not exist
         if not current_record:
             record = DME.createRecord(DME.prepareRecord(new_record))
-            monitor = DME.updateMonitor(record['id'], DME.prepareMonitor(new_monitor))
```

Comment:
Why are you doing this change? I don't know how that API works, but `new_monitor` might not contain the full new state of the monitor (assuming the API only makes changes specified in `new_monitor` and keeps other values the same).